### PR TITLE
Add health-o-meter

### PR DESCRIPTION
**Pull request by deborahleehamel**
add bootstrap
add bootstrap progress bar
add jQuery plugin for webpack
add draft instructions panel
add draft start button

All Tests Passing - 58

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -119,4 +122,10 @@ function recordFoodScore(foodsArray, food){
   foodsArray.shift(food);
   total += newScore;
   document.getElementById('score-game').innerHTML = "Score: " + total;
+  var totalPointsBar = document.getElementById('total-points');
+  var winPointsBar = document.getElementById('win-points');
+  totalPointsBar.style = "width:" + (total / 10) + "%";
+  totalPointsBar.innerHTML = total + "Health points";
+  winPointsBar.style = "width:" + (1000 - (total / 10)) + "%";
+  winPointsBar.innerHTML = (1000 - total) + "Needed to win";
```
Comment (chompasina) [inline lib/index.js:135]: I know we're just still working on this formatting but looks great so far!

File: style.css [hunk 2]
```diff
@@ -3,4 +3,23 @@
...
+.container { margin-top: 10px; }
+
+/*.progress-bar-vertical {
+  width: 20px;
+  min-height: 100px;
+  display: flex;
+  align-items: flex-end;
+  margin-right: 20px;
+  float: left;
+}
```
Comment (chompasina) [inline style.css]: We're probably good to remove this later :)

File: webpack.config.js [hunk 3]
```diff
@@ -14,10 +15,20 @@ module.exports = {
...
   resolve: {
     extensions: ['', '.js', '.json', '.scss', '.css', '.png']
-  }
+  },
+  plugins: [
+    new webpack.ProvidePlugin({
+     $: "jquery",
+     jQuery: "jquery"
+   })
+  ]
```
Comment (chompasina) [inline webpack.config.js:33]: Yay! Now we have jquery!

File: package.json [hunk 4]
```diff
@@ -36,5 +36,13 @@
     "url-loader": "^0.5.7",
     "webpack": "^1.10.1",
     "webpack-dev-server": "^1.10.1"
+  },
+  "dependencies": {
+    "bootstrap-webpack": "0.0.5",
+    "exports-loader": "^0.6.3",
```
Comment (karinamzalez) [inline package.json:42]: thank you for removing the bootstrap CDNs and using webpack instead!

**Comment by chompasina** [comment]
Looks great!

**Pull request merged**